src/app.js

```diff
@@ -87,15 +87,15 @@ $status_text.default();
 var $toolbox = $ToolBox();
 var $colorbox = $ColorBox();
 
+if(window.file_entry){
+	open_from_FileEntry(window.file_entry);
+}
+
 reset_file();
```

Comment:
I'm curious, how does this work if it does `reset_file` right afterwards? Does it load the image but lose the filename?
Also, what's the reason for moving it?

Reply:
Without moving it, `file_entry` is `null` from `reset_file()` when the app is launched, so the files fail to open. After moving it, all of the files open successfully. From your code, `file_entry` is only really needed for the initial loading.